study.md

```diff
@@ -21,18 +21,17 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+<!--Just want to clarify on callback functions, does it just mean a nested function? Like if, in math terms, I wanted to solve f(g(x)) is g(x) the callback function? And if it is, am I allowed to call g(x) by itself at any point after that or can I always call g(x) back inside another function?-->
```

Comment:
Nested functions are not the same as callback functions.  We'll discuss nested functions when we go over closures later on.